.github/workflows/rust.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     - name: Install MSRV
       uses: actions-rs/toolchain@v1
       with:
-        toolchain: 1.57.0
+        toolchain: 1.63.0
```

Comment:
Hey! Sorry for the delayed review.

Do we need 1.63 for this change, or can we get by with 1.57? I'm trying to keep the MSRV around a year old or so.

Reply:
I don't think so, can try reverting this. Besides that, is everything all good?

Reply:
I just commented about the ublox_cli changes, but then yeah everything looks good. (it looks like there might be some merge conflicts, I haven't gotten a chance to look at them yet)